src/components/Enum4Linux/Enum4Linux.tsx

```diff
@@ -9,20 +9,20 @@ import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFil
 
 const title = "Enum4Linux";
 const description_userguide =
-    "Enum4linux is a tool used for the enumeration of information from Windows and Samba operating systems. " +
+    "Enum4linux is a tool used for enumerating information from Windows and Samba operating systems. " +
     "It is particularly useful for identifying the remote OS of a system and providing a list of the users " +
     "and group memberships found within the system.\n\nOptions for the tool can be found at: " +
     "https://www.kali.org/tools/enum4linux/\n\n" +
     "Using Enum4Linux:\n" +
-    "Step 1: Enter a Target IP address.\n" +
+    "Step 1: Enter a target IP address.\n" +
     "       Eg: 192.168.1.1\n\n" +
```

Comment:
Make in line with line 17. Remove \n\n after the example and simply have it read:

Step 1: Enter a target IP address. E.g 192.168.1.1\n" +